charts/kube-proxy-2.0.0/values.yaml

```diff
@@ -0,0 +1,97 @@
+image:
+  repository: container-registry.oracle.com/olcne/kube-proxy
+  pullPolicy: IfNotPresent
+  tag: "current"
```

Comment:
Is the tag `current` correct? is it some typo?

Reply:
May be change to `repository: olcne/kube-proxy`?

Reply:
In new builds of OCK, the kube-proxy image that is baked into the image is always tagged with "current".  That way we can include the container image in the OS and not have to worry about coordinating tags between versions.  The "current" tag and fully qualified container image name are both intentional and required.